semantic-tags/src/Tags/Tag.hs

```diff
@@ -23,5 +23,7 @@ data Kind
   | Module
   -- References
   | Call
+  -- Types
+  | Type
```

Comment:
This looks reasonable to me, but do we have to update an enum in a .proto file somewhere to account for this new case?

Reply:
It's just a [string in the proto](https://github.com/github/semantic/blob/master/proto/semantic.proto#L166) right now so this is 👍